src/main/kotlin/org/taymyr/lagom/elasticsearch/search/dsl/query/term/TermLevelQuery.kt

```diff
@@ -0,0 +1,8 @@
+package org.taymyr.lagom.elasticsearch.search.dsl.query.term
+
+import org.taymyr.lagom.elasticsearch.search.dsl.query.Query
+
+/**
+ * @author Evgeny Stankevich {@literal <estankevich@fil-it.ru>}.
+ */
+interface TermLevelQuery : Query
```

Comment:
`IdsQuery` should be moved to `term` package and extended from `TermLevelQuery`.